core/src/main/java/com/sequenceiq/cloudbreak/controller/ImageCatalogV1Controller.java

```diff
@@ -123,7 +124,9 @@ private ImageCatalogResponse convert(ImageCatalog imageCatalog) {
     }
 
     private List<ImageCatalogResponse> getAll() {
-        return toJsonList(imageCatalogService.listForUsersDefaultOrganization(), ImageCatalogResponse.class);
+        Set<ImageCatalog> imageCatalogs = imageCatalogService.listForUsersDefaultOrganization();
+        imageCatalogs.add(imageCatalogService.getCloudbreakDefaultImageCatalog());
```

Comment:
I think it should be moved to the service layer and then it wouldn't be implemented twice for the 2 APIs